src/Details/Details.js

```diff
@@ -1,11 +1,24 @@
-import React from 'react';
+import React, { useState } from 'react';
 import { Link } from "react-router-dom";
 
 export default function Details(props) {
+    const [message, setMessage] = useState('');
+
+    React.useEffect(
+        () => {
+            const timedMsg = setTimeout(() => setMessage('Coming Soon! :-P'), 3000)
+            return () => clearTimeout(timedMsg)
```

Comment:
Mmm why is this return statement necessary?

Reply:
because I accidently used a clearInterval at first - mixed up timeout and interval.
I will delete this!